docker/models/containers.py

```diff
@@ -173,9 +173,10 @@ def exec_run(self, cmd, stdout=True, stderr=True, stdin=False, tty=False,
                 exit_code: (int):
                     Exit code for the executed command or ``None`` if
                     either ``stream```or ``socket`` is ``True``.
-                output: (generator or bytes):
+                output: (generator or bytes or tuple):
```

Comment:
nit: Use `Oxford comma` to be clear please.
Like: `output: (generator, bytes, or tuple):`

Reply:
Okay, Thank you for point this out! I've changed it.